src/main/java/com/solvd/bank/domain/Account.java

```diff
@@ -10,13 +12,20 @@
 
 @XmlRootElement(name = "account")
 @XmlType(propOrder = {"id", "branch", "customer", "amount", "dateCreated", "holds"})
+@JsonRootName("account")
 public class Account {
 
+    @JsonProperty("id")
     private int id;
+    @JsonProperty("branch")
     private Branch branch;

```

Comment:
please add an empty line between variables
```suggestion
    @JsonProperty("id")
    private int id;
    
    @JsonProperty("branch")
    private Branch branch;
    

```